tests/acceptance/features/bootstrap/TwoFactorTOTPContext.php

```diff
@@ -250,6 +250,104 @@ public function theSecretCodeFromQRCodeShouldMatchWithTheOneDisplayedOnTheWebUI(
 		);
 	}
 
+	/**
+	 * Send request with secet key for two factor authentication
+	 *
+	 * @param string $user
+	 * @param string $secretKey
+	 *
+	 * @return void
+	 */
+	public function sendRequestWithSecretKey($user, $secretKey) {
+		$response = \TestHelpers\OcsApiHelper::sendRequest(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getAdminUsername(),
+			$this->featureContext->getAdminPassword(),
+			'GET',
+			"/apps/twofactor_totp/api/v1/validate/$user/$secretKey",
+			$body = [],
+			$ocsApiVersion = 1
+		);
+		$this->featureContext->setResponse($response);
+	}
+
+	/**
+	 * @When the administrator tries to verify the otp key for user :user using the correct key
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function theAdministratorTriesToVerifyTheOtpKeyForUserUsingTheCorrectKey($user) {
+		$secretKey = $this->generateTOTPKey();
+		$this->sendRequestWithSecretKey($user, $secretKey);
+	}
+
+	/**
+	 * @Then the result of the last verification request should be true
+	 *
+	 * @throws \Exception
+	 *
+	 * @return void
+	 */
+	public function theResultOfTheLastVerificationRequestShouldBeTrue() {
+		$result = \json_decode(
+			\json_encode(
+				$this->featureContext->getResponseXml()->data[0]
+			),
+			true
+		)['result'];
+		if ((int)$result !== 1) {
+			throw new \Exception(
+				"Invalid OTP"
+			);
+		}
+	}
+
+	/**
+	 * @When the administrator tries to verify the otp key for user :user using the wrong key
```

Comment:
```suggestion
	 * @When the administrator tries to verify with the random key :key for user :user
```

Reply:
Or, maybe this:
```suggestion
	 * @When the administrator tries to verify with the random key for user :bust_in_silhouette: 
```
I do prefer the first one though.